### ISSUE TITLE
Filtering the “next solutions to mentor”

### ISSUE THREAD

**Issue by NobbZ** [exercism/v2-feedback#195]
There are filters on the mentors dashboard (https://mentors-beta.exercism.io/mentor/dashboard) which do not seem to apply to "Next solutions to mentor".

In my case where I do not have any pending solutions, the do not do anything on a first glance. Only on a second try I realized, that “You currently have no solutions that need mentoring” was replaced by “You have no solutions in this category”.

It would be nice if the filters would also apply to the “next solutions to mentor”.

Especially when doing initial mentoring, I prefer to filter by exercise, such that I can batch-mentor all open `hello-world` in one go instead instead of doing them randomly.

**Comment by hilary** [comment]
I'm assuming that once I provide feedback to a student (comment on a solution), the solution should transition to status 'Waiting for them'. After I comment on a solution, the solution disappears from the list of "Next solutions to mentor", however, the filter still says I have no solutions in this category for that status when I set the status to 'Waiting for them'. I'm not sure if there's a bug in the software or in my understanding of how the process should work...

Just FYI, approving a solution removes it from the list of "Next solutions to mentor" that I see on the dashboard, which is what I would expect.

**Comment by jpreese** [comment]
Also agree that it would be nice for filtering "Next solutions to mentor" 👍

**Comment by scarerkite** [comment]
~Came here to report the same as hilary - I've commented on 4 solutions, and they're not showing up when I filter to 'Waiting for them'~

My solutions that I thought should be in 'Waiting for them' were actually in 'Stale', as suggested below, which is covered by a different bug.

**Comment by NobbZ** [comment]
I have some in waiting for them, but also I found a solution I mentored minutes ago in stale. The solution itself is 15 days old.

Stale status should be given on age of the last interaction with the solution, not based on the age of the solution.

But thats not what this issue is about. This issue should track the inability to filter the "Next solutions to mentor" by track and exercise.

**Comment by hilary** [comment]
A bit more info on filtering "next solutions to mentor."

If I have not yet mentored any solutions since I logged in to exercism (chrome 67.0.3396.99, 64-bit on macOS 10.13.5), using the 'Track' filter kind of works. I say kind of, because only solutions from the selected language appear, however, the "You have no solutions in this category." info box displays below the filters.

Once I have mentored a solution, the 'Track' filter stops working entirely.
